web/src/app/modules/sugarloaf/components/smart/uploader/uploader.component.ts

```diff
@@ -3,7 +3,7 @@
 //
 
 import { Component, OnDestroy, OnInit } from '@angular/core';
-import { WebsocketService } from '../../../../shared/services/websocket/websocket.service';
+import { WebsocketService } from '../../../../../data/services/websocket/websocket.service';
 import { Subscription } from 'rxjs';
 
 @Component({
```

Comment:
Can `WindowToken` be injected here as well? It can allow writing tests for `upload` in the future

Reply:
It should be. I was trying to limit the scope of this change. It should also be updated in RelativePipe, HeptagonLabel, in the getAPIBase code. I created #1468 to track this.